automation_tools/scripts/ga-migration/main.py

```diff
@@ -135,6 +136,12 @@ def migrate_repo(path):
     # Remove bak files
     delete_file(path + "*.bak")
 
+    # Add newline at EOF
+    add_nl_at_eof(path + ".github/workflows/tests.yml")
+    add_nl_at_eof(path + "MANIFEST.in")
```

Comment:
Moderate: [This statement is already taking the EOF into account](https://github.com/inveniosoftware/automation-tools/blob/c708834af8540845c5084d6b7e4c4d440f46b183/automation_tools/scripts/ga-migration/main.py#L95), this would add a double EOF.